test/integration/targets/nxos_nxapi_transports/tasks/nxapi.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- block:
+
+  - meta: end_play
```

Comment:
@trishnaguha We added this `end_play` so that it will not run in your CI but ideally if we could update your group_vars with the following then we can remove this check.

```
nxapi_ssl:
  host: "{{ ansible_ssh_host }}"
  username: "{{ nxos_nxapi_user | default('admin') }}"
  password: "{{ nxos_nxapi_pass | default('password') }}"
  transport: nxapi
  use_ssl: yes
  validate_certs: False
```